middleware.ts

```diff
@@ -11,17 +11,22 @@ export const middleware: NextMiddleware = ( req: NextRequest ) => {
 	// https://github.com/vercel/next.js/pull/45772
 	// https://github.com/vercel/next.js/blob/v13.1.1/packages/next/server/base-server.ts#L1069
 	const headers = new Headers(req.headers);
-  headers.delete('x-middleware-prefetch');
+	headers.delete('x-middleware-prefetch');
 
 	// Required health check endpoint on VIP. Do not remove.
 	if ( req.nextUrl.pathname === '/cache-healthcheck' ) {
 		return NextResponse.rewrite( new URL( '/api/healthcheck', req.url ) );
 	}
 
 	// Continue as normal through the Next.js lifecycle.
-	return NextResponse.next({
-    request: {
-      headers,
-    },
-  });
+	const response = NextResponse.next( {
+		request: {
+			headers,
+		},
+	} );
+
+	// WPVIP error pages https://docs.wpvip.com/infrastructure/edge-servers/error-pages
```

Comment:
Can there be a little more explanation of what this header is for and why it is set here?

Reply:
Done in 88fee8a34c61e49ecc3ede5b3482e74ae74a66ce